addons/apecloud-mysql/templates/cmpd-proxy.yaml

```diff
@@ -181,4 +206,73 @@ spec:
       valueFrom:
         serviceRefVarRef:
           name: etcd
-          endpoint: Required
\ No newline at end of file
+          endpoint: Required
+          optional: true
+    - name: ETCD_LOCAL_POD_LIST
+      valueFrom:
+        componentVarRef:
+          compDef: {{ include "apecloud-etcd.componentDefName" . }}
+          optional: true
+          instanceNames: Optional
+    - name: ETCD_LOCAL_HEADLESS
+      valueFrom:
+        serviceVarRef:
+          compDef: {{ include "apecloud-etcd.componentDefName" . }}
+          name: headless
+          optional: true
+          host: Optional
+    - name: ETCD_LOCAL_PORT
+      valueFrom:
+        serviceVarRef:
+          compDef: {{ include "apecloud-etcd.componentDefName" . }}
+          name: etcd
+          optional: true
+          port:
+            name: etcd-client
+            option: Optional
+---
+apiVersion: apps.kubeblocks.io/v1alpha1
```

Comment:
add replicas llimits to this cmpd, it is a simplified version with only ONE replica